CONTRIBUTING.md

```diff
@@ -160,6 +160,34 @@ Where again, `<PATTERN>` is the name of the folder listed in the
 [test/integration/tests/](https://github.com/commercialhaskell/stack/tree/master/test/integration/tests)
 folder.
 
+## CI Build rules
+
+We use [Azure](https://dev.azure.com/commercialhaskell/stack/_build)
+to do CI builds on Stack. There are two types of build which happens
+there:
+
+### Unit based build
+
+This build runs only the unit tests in the Stack codebase. These are
```

Comment:
It does more than that: it builds the code with `--pedantic`, it runs hlint checks, _and_ it runs all test suites on multiple GHC/OS configurations.

Reply:
I have added that info now.